### ISSUE TITLE
Allow whitespaces on comand parameter

### ISSUE THREAD

**Issue by JGeek00** [jeffrey0606/flutter_to_debian#20]
Hi, will explain my situation.

The executable file of my application has whitespaces on it's name, I have to set that name (with it's whitespaces) on the command parameter of the debian.yaml file, but if I add the name with the whitespaces and I compile it, when I install the application, the entry does not appear on the launcher. I tried wrapping the name with "" and '' on the command parameter (ex: "my app name" or 'my app name'), I also tried escaping the whitespace (my\ app\ name) but nothing works. The only way that I have to make it to work is to remove the whitespaces on the executable name and add that name without the whitespaces on the command parameter, but I don't want to do that because the app name when installed wouldn't have it's whitespaces.

**Comment by jeffrey0606** [comment]
Hello @JGeek00 , Sorry I missed this.
Please do you still need help with this ?

**Comment by JGeek00** [comment]
Yeah, I would like to have whitespaces on the app name, but don't know if it's possible.

**Comment by jeffrey0606** [comment]
Hi @Gustl22 , sorry to bother you. Please can you have a look at this issue? 🙏 
Currently not on a Linux OS to implement and test.

**Comment by Gustl22** [comment]
I don't quite understand, @JGeek00 you shouldn't be able to generate a file with whitespace in the first place, as flutter applications don't allow that in their project names: https://dart.dev/tools/pub/pubspec#name

And further the application executable name (here: `command`) is different from the actual name written on the launcher, which is defined in the `.desktop` file, which does allow whitespaces :)

**Comment by JGeek00** [comment]
@Gustl22 the problem I have is that on the GNOME launcher, the app name is displayed correctly with it's spaces as it's configured on the .desktop file, but on the appbar the name is displayed without its spaces.
![image](https://github.com/jeffrey0606/flutter_to_debian/assets/47545344/96db7853-b6b6-469c-8866-0f3d17673611)
![image](https://github.com/jeffrey0606/flutter_to_debian/assets/47545344/e8c49504-17b6-4355-bcdc-e83a06307f6d)

**Comment by Gustl22** [comment]
@JGeek00 I don't know what OS you use, but I can tell that on GNOME 42.9 on Ubuntu 22.04 it uses the `Name` property of the `.desktop` file.

You shouldn't need to change the binary name or even remove the chars.

![Screenshot from 2023-07-26 20-08-26](https://github.com/jeffrey0606/flutter_to_debian/assets/18537755/7088d632-65cc-4df3-a5b3-9b4736b24e5a)

**Comment by JGeek00** [comment]
@Gustl22 this is the configuration i'm using

.desktop file
```
[Desktop Entry]
Name=AdGuard Home Manager
GenericName=AdGuard Home Manager
Comment=Manage your AdGuard Home server
Exec=adguard-home-manager
Icon=${SNAP}/meta/gui/adguard-home-manager.png
Terminal=false
Type=Application
Categories=Utilities;
```


debian.yaml
```
flutter_app: 
  command: AdGuardHomeManager
  arch: x64
  parent: /usr/local/lib

control:
  Package: AdGuardHomeManager
  Version: 2.6.0
  Architecture: amd64
  Essential: no
  Priority: optional
  Depends:
  Maintainer: JGeek00
  Description: AdGuard Home control app
```

**Comment by Gustl22** [comment]
How do you get to [this](https://user-images.githubusercontent.com/47545344/246119628-e8c49504-17b6-4355-bcdc-e83a06307f6d.jpeg) view. Haven't seen that on my pc. Did you install a custom launcher?

**Comment by JGeek00** [comment]
Mmmmm no. That comes by default with GNOME. There it appears the icon and name of the app that you are currently using. For example with Chrome. And when you click on it it shows you a menu.
![image](https://github.com/jeffrey0606/flutter_to_debian/assets/47545344/7ad65c30-8475-45d4-abbb-015fc77c6a49)
![image](https://github.com/jeffrey0606/flutter_to_debian/assets/47545344/ee627bc6-5145-44c8-96cd-00b61a014fd8)

**Comment by Gustl22** [comment]
Tested once again and it still takes the name of the `.desktop` file from `/usr/share/applications/`. Maybe you messed sth up during installation. Check if the file exists at the path.

![Screenshot from 2023-08-14 21-15-24](https://github.com/jeffrey0606/flutter_to_debian/assets/18537755/363b4e61-72c7-4a58-b3b6-637bd378261e)
